src/components/SiteHeader/SiteHeader.jsx

```diff
@@ -16,20 +16,45 @@ const HEADER_HIDDEN = {
 /**
  * JSX element for Barnes logos.
  */
-const Logo = ({ size, width, height }) => (
-  <svg className={`a-logo__svg a-logo__svg--${size}`} width={width} height={height} aria-labelledby='logo-title'>
+const Logo = ({ size, width, height, logo, className }) => (
+  <svg className={`a-logo__svg a-logo__svg--${size} ${className ? className : ""}`} width={width} height={height} aria-labelledby='logo-title'>
     <title id='logo-title'>Barnes</title>
-    <use xlinkHref={`#icon--logo-${size}`}></use>
+    <use xlinkHref={`#icon--logo-${logo ? logo : size}`}></use>
   </svg>
 );
 
+
 // Static info mapped to static Logo JSX element.
-const LOGOS = [
+const logoSizes = [
   { size: 's', width: 121, height: 37 },
   { size: 'm', width: 146, height: 45 },
   { size: 'l', width: 164.958, height: 50 },
   { size: 'xl', width: 200, height: 62 },
-].map((logo => <Logo key={logo.size} {...logo} />));
+];
+
+const logo100Sizes = [
+  {size: 'xs', width: 121, logo: "100"},
+  { size: 's', height: 37, logo: "100H" },
+  { size: 'm', height: 45, logo: "100H" },
+  { size: 'l', width: 164.958, logo: "100" },
+  { size: 'xl', width: 200, logo: "100" },
+];
+
+const Logos = ({ ref, tabIndex, isCentennial}) => (
+  <a
+    className='a-logo g-header__logo'
+    href={MAIN_WEBSITE_DOMAIN}
+    tabIndex={tabIndex}
+    ref={ref}
+  >
+    {isCentennial ? (
```

Comment:
A really easy way to test that everything still looks good w/ the standard logo is to switch this to `!isCentennial`

Reply:
yup!